### ISSUE TITLE
`deno repl` is hanging after upgrading to 1.32.2

### ISSUE THREAD

**Issue by roj1512** [denoland/deno#18540]
```
deno 1.32.2 (release, x86_64-unknown-linux-gnu)
v8 11.2.214.9
typescript 5.0.3
```

**Comment by dsherret** [comment]
Does this occur only when launching in specific folders?

**Comment by dsherret** [comment]
Pretty sure this is a duplicate of #18538

**Comment by roj1512** [comment]
Yes.

**Comment by roj1512** [comment]
> Does this occur only when launching in specific folders?

Yes.

**Comment by roj1512** [comment]
> Pretty sure this is a duplicate of https://github.com/denoland/deno/issues/18538

Maybe edit its title to exclude Windows and generalize it?

**Comment by dsherret** [comment]
Should be fixed now in 1.32.3.

**Comment by roj1512** [comment]
Target clear. Copy.
